rocketpy/environment/environment.py

```diff
@@ -276,15 +276,21 @@ def __init__(
         timezone="UTC",
         max_expected_height=80000.0,
     ):
```

Comment:
I can't comment on some lines because they were not edited (thanks GitHub).

Here are some points I would like to mention:
* in the date description, does it really mean that the hour is ALWAYS in UTC? What if I use a different timezone? Is my inputted time gonna be converted or not?
* Please mention the default values for latitude and longitude
* Sometimes it says "timezone", and sometimes "time zone" . I searched but could not find the "correct way". But we should keep a standard imo.
* in the datum description, it is not saying that the argument is optional. It should.

Reply:
Great catches, those should be fixed by commit [27dd73d](https://github.com/RocketPy-Team/RocketPy/pull/565/commits/27dd73d24fa82f3cbf066d6edb5aa3063b102099).